src/test/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/ReferenceConfidenceModelUnitTest.java

```diff
@@ -106,38 +277,66 @@ public void setupModel() {
                     for ( int i = 0; i < anchor.length(); i++ ) expected.add(1);
                     for ( int i = 0; i < repeat.length(); i++ ) expected.add(readLen == repeat.length() ? 1 : 0);
                     for ( int i = 0; i < anchor.length(); i++ ) expected.add(0);
-                    tests.add(new Object[]{read, readCigar, ref, repeatUnit.length(), expected});
+                    tests.add(new Object[]{read, readCigar, null, ref, repeatUnit.length(), 0, expected});
 
                     final List<Integer> result = new ArrayList<>(Collections.nCopies(ref.length() - anchor.length(), 1));
                     result.addAll(Collections.nCopies(anchor.length(), 0));
-                    tests.add(new Object[]{ref, refCigar, ref, repeatUnit.length(), result});
+                    tests.add(new Object[]{ref, refCigar, null, ref, repeatUnit.length(), 0, result});
                 }
             }
         }
 
+        {//test that the behavior is still confusing and wrong in the case where read.length() disagrees with the realigned read length due to indels (caught in a debugger)
```

Comment:
Replace "test that the behavior is still confusing and wrong" with a description of the actual behavior we're testing for.